### PR TITLE
Small formatting fixes and fixes from the minimalism sessions for 4.2.

### PR DESCRIPTION
**Pull request by yhontyk**
Changes from https://github.com/openshift/openshift-docs/pull/19392 for 4.2 branch.

@bergerhoffer

### CODE REVIEW

File: cli_reference/openshift_developer_cli/understanding-odo.adoc [hunk 1]
```diff
@@ -30,12 +30,12 @@ Application::
 An application is a program designed for end users. An application consists of multiple microservices or components that work individually to build the entire application.
 Examples of applications: a video game, a media player, a web browser.
 Component::
-A component is a building block of an application. Typically it is a software package, a web service, a web resource, or a module that encapsulates a set of related functions or data.
-Examples of components: Node.js, Perl, PHP, Python, Ruby.
+A component is a building block of an application. Typically it is a software package, a web service, a web resource, or a module that encapsulates a set of related functions or data. 
+Examples of components: Node.js, Pearl, PHP, Python, Ruby.
```
Comment (bergerhoffer) [inline cli_reference/openshift_developer_cli/understanding-odo.adoc]: Should still be Perl

**Review by bergerhoffer — COMMENTED** [review]
Just one typo to fix, then I can merge!

**Comment by boczkowska** [comment]
@bergerhoffer fixed! Thank you!

**Comment by bergerhoffer** [comment]
Thanks, looks good now!

**Pull request merged**